cp3pt0-deployment/common/utils.sh

```diff
@@ -862,4 +862,17 @@ function scale_up() {
         msg "Scaling up operand-deployment-lifecycle-manager deployment in ${operator_ns} namespace back to 1"
         scale_deployment ${operator_ns} operand-deployment-lifecycle-manager 1
     fi
+}
+
+function accept_license() {
+    kind=$1
+    namespace=$2
```

Comment:
```suggestion
    local kind=$1
    local namespace=$2
```
localize new variables in functions

Reply:
suggest also add a third variable `local cr_name=$3`, since we already know the names of CRs to check from installation and upgrade perspective. No point in running an unnecessary get. 

Just try to patch the CR, and if it fails, then return error, so that wrappers can handle and re-run script if needed